examples/03_connectivity/plot_atlas_comparison.py

```diff
@@ -59,13 +60,11 @@
 from nilearn import plotting
 
 # create masker to extract functional data within atlas parcels
-masker = NiftiLabelsMasker(labels_img=yeo['thick_17'], standardize=True,
-                           memory='nilearn_cache')
+masker = MultiNiftiLabelsMasker(labels_img=yeo['thick_17'], standardize=True,
```

Comment:
Through a comment, make it explicit that you use `MultiNiftiLabelsMasker`, so that people notice and understand why.